tests/test_orchestrator.py

```
@@ -33,140 +37,101 @@ def mock_orchestrator():
 class TestHealthEndpoint:
     """Test health check endpoint"""
 
-    @patch('server.orchestrator.config.validate')
-    @patch('os.path.exists', return_value=True)
-    @patch('os.access', return_value=True)
-    def test_health_returns_200_healthy(self, mock_access, mock_exists, mock_validate, client):
+    @patch('server.app.state.kimi_client.health_check', return_value=True)
+    def test_health_returns_200_healthy(self, mock_health_check, client):
```

Ruff (0.14.10)

Unused method argument: `mock_health_check`

(ARG002)